src/test/index.js

```diff
@@ -2,34 +2,45 @@
 
 const pmap = require('p-map')
 const _ = require('lodash')
+const path = require('path')
 
 const node = require('./node')
 const browser = require('./browser')
 
 const userConfig = require('../config/user')
 
+const taskEnabled = (ctx, name) => _.includes(ctx.target, name)
+
 const TASKS = [{
+  id: 'node',
   title: 'Test Node.js',
-  task: node,
-  enabled: (ctx) => _.includes(ctx.target, 'node')
+  task: node
 }, {
+  id: 'browser',
   title: 'Test Browser',
-  task: browser.default,
-  enabled: (ctx) => _.includes(ctx.target, 'browser')
+  task: browser.default
 }, {
+  id: 'webworker',
   title: 'Test Webworker',
-  task: browser.webworker,
-  enabled: (ctx) => _.includes(ctx.target, 'webworker')
+  task: browser.webworker
 }]
 
 module.exports = {
   run (opts) {
+    const pkg = require(path.join(process.cwd(), 'package.json'))
```

Comment:
This should be part of `utils.getUserConfig` where it uses both `.aegir` and `package.json` files to populate the configuration.